Form/Type/ApiCommentType.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NewsBundle\Form\Type;
+
+use Sonata\CoreBundle\Form\Type\BaseDoctrineORMSerializationType;
+
+class ApiCommentType extends BaseDoctrineORMSerializationType
```

Comment:
`@package` is not used according to the Symfony convention.

And "Class ApiCommentType" is not really useful. :stuck_out_tongue: 

The `@author` tag should be enough. :wink: